utils/zerovec/src/zerovec/slice.rs

```diff
@@ -251,10 +251,10 @@ where
     ///     }
     /// };
     ///
-    /// let zs_char: &ZeroSlice<char> = zs_u32.try_as_converted().expect("valid code points");
+    /// let zs_i32: &ZeroSlice<i32> = zs_u32.try_as_converted().expect("valid code points");
     ///
-    /// assert_eq!(zs_u32.get(0), Some(u32::from('🍿')));
-    /// assert_eq!(zs_char.get(0), Some('🍿'));
+    /// assert_eq!(zs_u32.get(0), Some(127871));
+    /// assert_eq!(zs_i32.get(0), Some(127871));
```

Comment:
Nit: Choose a bit pattern that differs between u32 and i32 (or go back to chars)

Reply:
Updated to char, [u8; 3]